packages/x-date-pickers-pro/src/DesktopDateRangePicker/DesktopDateRangePicker.test.tsx

```diff
@@ -553,6 +553,7 @@ describe('<DesktopDateRangePicker />', () => {
           onAccept={onAccept}
           onClose={onClose}
           slotProps={{ actionBar: { actions: ['clear'] } }}
+          value={[null, null]}
```

Comment:
Behavior change n°2

Reply:
Not using it makes sure that we don't have some `===` hidden somewhere to do the equality check instead of a deep comparison.